packages/character-count-ui/src/CharacterCounter.tsx

```diff
@@ -24,13 +24,14 @@ const calculateCharCount = (editor?: TEditor) => {
   return toArray(getTextString(editor)).length
 }
 
-export function getCharacterCount(id: string): number {
```

Comment:
this hook has been removed as it was being used in `core/dreifuss` but that caused incorrect behavior. So I just made getting char count by a prop (callback function) to the Component (`CharCountToolbar`).